tower-balance/src/lib.rs

```diff
@@ -151,7 +151,7 @@ where
         debug!("promoting to ready: {}", n);
         // Iterate through the not-ready endpoints from right to left to prevent removals
         // from reordering services in a way that could prevent a service from being polled.
-        for idx in (0..n-1).rev() {
```

Comment:
i think this would be slightly clearer as:

```yaml
if n == 0 {
  return Ok(());
}
for idx in (0..n-1).rev() { 
  ...
}
```

it makes the intent a bit clearer...

Reply:
Sorry, I don't follow? This is the line that was removed.